static/js/components/EducationDisplay.js

```diff
@@ -61,18 +61,20 @@ export default class EducationDisplay extends ProfileFormFields {
     );
     return (
       <Cell col={12} className="profile-form-row" key={index}>
-        <div className="basic-info">
+        <div className="basic-info full-width border-bottom">
           <div className="profile-row-name">
-            <span className="school-type">
+            <div className="school-type">
               { degree }
-            </span><br/>
+            </div><br/>
             { entry.school_name }
           </div>
+        </div>
+        <div className="basic-info full-width border-bottom education-top-spacing">
```

Comment:
I think here you should just put another class on this div to indicate it's a userpage row (like `.user-page-row` or something).

then all the styling for adding the border at the bottom and doing layout can be just on that selector.